Maquetado/scss/titles.scss

```diff
@@ -0,0 +1,18 @@
+.title {  
+
+  &:before {
```

Comment:
use `::` instead. `:` is for pseudo-classes and `::` for pseudo-elements, it works with `:` because chrome is a good guy but it's not supported in all browsers